src/moai-sim/MOAIDeck.cpp

```diff
@@ -133,15 +133,14 @@ MOAIDeck::~MOAIDeck () {
 
 //----------------------------------------------------------------//
 bool MOAIDeck::Overlap ( u32 idx, const ZLVec2D& vec, u32 granularity, ZLBounds* result ) {
-
-	return this->Overlap ( idx, vec, granularity, result );
```

Comment:
@patrickmeehan did this ever work? am I missing some magic overload logic? this looks like it will always be recursive. is it right to just return false instead? or is there another overlap method that this should be delegating to.

Reply:
Looks like I didn't update it to MOAIDeck_Overlap (). I'm bringing in a new naming convention for virtual methods as I refactor/rewrite, but the public-facing methods keep their original names. So a few glitches like this that would result in a recursive call.